### PR TITLE
Xification the algorithm code

### PR DESCRIPTION
**Pull request by tridactyla**
To continue with the glorious spirit of change of our dear Elon Musk, I introduce you to "Xification the algorithm code", the definitive step towards the death of the Twitter brand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# Twitter's Recommendation Algorithm
+# X's Recommendation Algorithm
 
-Twitter's Recommendation Algorithm is a set of services and jobs that are responsible for serving feeds of Tweets and other content across all Twitter product surfaces (e.g. For You Timeline, Search, Explore, Notifications). For an introduction to how the algorithm works, please refer to our [engineering blog](https://blog.twitter.com/engineering/en_us/topics/open-source/2023/twitter-recommendation-algorithm).
+X's Recommendation Algorithm is a set of services and jobs that are responsible for serving feeds of Tweets and other content across all X product surfaces (e.g. For You Timeline, Search, Explore, Notifications). For an introduction to how the algorithm works, please refer to our [engineering blog](https://blog.X.com/engineering/en_us/topics/open-source/2023/X-recommendation-algorithm).
```
Comment (jasonycw) [inline README.md:3]: There are no `Tweets` but `x's` or `Xs`
Ref: https://twitter.com/elonmusk/status/1683365872323133442

File: cr-mixer/server/src/main/scala/com/X/cr_mixer/filter/RetweetFilter.scala [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import com.X.util.Future
+
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/***
+ * Filters candidates that are retweets
+ */
+@Singleton
+case class RetweetFilter @Inject() () extends FilterBase {
```
Comment (jasonycw) [inline cr-mixer/server/src/main/scala/com/X/cr_mixer/filter/RetweetFilter.scala:15]: There is no `Retweets`, but `Reposts`
Ref: https://twitter.com/PopBase/status/1685320104085757952

File: README.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 
 ## Architecture
 
-Product surfaces at Twitter are built on a shared set of data, models, and software frameworks. The shared components included in this repository are listed below:
+Product surfaces at X are built on a shared set of data, models, and software frameworks. The shared components included in this repository are listed below:
 
 | Type | Component | Description |
 |------------|------------|------------|
 | Data | [tweetypie](tweetypie/server/README.md) | Core Tweet service that handles the reading and writing of Tweet data. |
```
Comment (jasonycw) [inline README.md:11]: There is no `Tweet` but `X`
Ref: https://twitter.com/elonmusk/status/1683055552404897792

File: README.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-|      | [unified-user-actions](unified_user_actions/README.md) | Real-time stream of user actions on Twitter. |
+|      | [unified-user-actions](unified_user_actions/README.md) | Real-time stream of user actions on X. |
 |      | [user-signal-service](user-signal-service/README.md) | Centralized platform to retrieve explicit (e.g. likes, replies) and implicit (e.g. profile visits, tweet clicks) user signals. |
```
Comment (jasonycw) [inline README.md:13]: But as a verb it should be `Post`
Ref: https://twitter.com/PopBase/status/1685171560121024512

**Review by jasonycw — COMMENTED** [review]
Still some more terminology needs to be 𝕏ify

File: README.md (hunk 1 above)
Comment (jasonycw) [reply README.md:3]: More changes needed, support `Xs` or `Posts`
https://twitter.com/elonmusk/status/1687559566001836032

File: README.md (hunk 3 above)
Comment (jasonycw) [reply README.md:11]: More changes needed, support `X` or `Post`
https://twitter.com/elonmusk/status/1687559566001836032

**Comment by pyxfluff** [comment]
This is a very poorly done find all replace all job. I will not be fixing it, and if I say why I'll be banned. The branding is incredibly stupid tldr, I'm still calling it Twitter for the rest of time. Hope a new CEO comes in and undoes this entire mess

**Review by pyxfluff — COMMENTED** [review]
it is twitter, not "x"

**Review by Debagnik — COMMENTED** [review]
the proper term would be Xify and not xification.

**Comment by everking** [comment]
> To continue with the glorious spirit of change of our dear Elon Musk, I introduce you to "Xification the algorithm code", the definitive step towards the death of the Twitter brand.

5,000 files? Too much for 1 PR?

**Comment by 1lordduck** [comment]
yeah this LGTM!

**Pull request closed without merging**